# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/udev/hotfix-modalias.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

For e.g x86 (but certainly other) cpu features we need to match mod alias'
however, the cpu has already the DRIVER set as processor. Fixing this to
match just like in newer systemd based udev by removing DRIVER != "?*".

--- udev-182/rules/80-drivers.rules.vanilla	2023-01-24 13:01:21.446081006 +0100
+++ udev-182/rules/80-drivers.rules	2023-01-24 13:04:07.476095547 +0100
@@ -2,7 +2,7 @@
 
 ACTION=="remove", GOTO="drivers_end"
 
-DRIVER!="?*", ENV{MODALIAS}=="?*", IMPORT{builtin}="kmod load $env{MODALIAS}"
+ENV{MODALIAS}=="?*", IMPORT{builtin}="kmod load $env{MODALIAS}"
 SUBSYSTEM=="tifm", ENV{TIFM_CARD_TYPE}=="SD", IMPORT{builtin}="kmod load tifm_sd"
 SUBSYSTEM=="tifm", ENV{TIFM_CARD_TYPE}=="MS", IMPORT{builtin}="kmod load tifm_ms"
 SUBSYSTEM=="memstick", IMPORT{builtin}="kmod load ms_block mspro_block"
